library.json

```diff
@@ -15,8 +15,8 @@
     }
   ],
   "dependencies": {
-    "name": "RCSwitch-CC1101-Driver-Lib",
-    "version": "https://github.com/LSatan/RCSwitch-CC1101-Driver-Lib.git#2bff72edbee1ee29cbd134d8b2fc47ba0dc0e5f0"
+    "name": "SmartRC-CC1101-Driver-Lib",
+    "version": "https://github.com/LSatan/SmartRC-CC1101-Driver-Lib"
```

Comment:
Could you please reference the version of the library you tested? I guess it is v.2.3.4 so change this line to "https://github.com/LSatan/RCSwitch-CC1101-Driver-Lib.git#v2.3.4"